CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing to WooCommerce Admin
+
+Hi! Thank you for your interest in contributing to WooCommerce Admin. We appreciate it.
+
+There are many ways to contribute – reporting bugs, feature suggestions, and fixing bugs.
+
+## Reporting Bugs, Asking Questions, Sending Suggestions
+
+Open [a GitHub issue](https://github.com/woocommerce/woocommerce-admin/issues/), that's all. If you want to prefix the title with a “Question:”, “Bug:”, or the general area of the application, that would be helpful but is no means mandatory. If you have write access, add the appropriate labels.
```

Comment:
Incorrect link here, should be `https://github.com/woocommerce/wc-admin/issues/new/choose`

Reply:
Also you can remove the prefix bit since @belcherj made some great templates for bugs vs feature requests.

Reply:
Fixed in https://github.com/woocommerce/wc-admin/pull/748/commits/8d827876244a21487add21a0e78ef8c879bcd300